src/ui/server/content-strings/pages/insurance/your-business/company/index.ts

```diff
@@ -5,6 +5,8 @@ const COMPANY_DETAILS = {
   WEBSITE: 'Enter your company website, if you have one (optional)',
   PHONE_NUMBER: 'Your UK telephone number (optional)',
   PHONE_NUMBER_HINT: 'We may need to contact you about your application',
+  ALTERNATIVE_TRADING_NAME: "What's your organisation's alternative trading name?",
+  ALTERNATIVE_TRADING_NAME_HINT: 'Your official trading name will still be on the policy.',
```

Comment:
Not a blocker since we have `PHONE_NUMBER_HINT`, all other hints are in field content strings instead of page content strings

Reply:
this whole page has everything in this file.  Needs a tech debt to move it